ODLA/include/ODLA/odla_common.h

```diff
@@ -135,6 +135,13 @@ typedef void odla_void;
 
 //! \brief Return status
 typedef enum {
+  ODLA_DL_ERROR,
```

Comment:
What is DL_ERROR ?
Add some doxygen comments for each error?

Reply:
maybe ODLA_UNSUPPORTED_OP?

Reply:
DL_ERROR  means encounter error(s) when dlopen a shared library